src/components/App/App.tsx

```diff
@@ -1,27 +1,27 @@
-import React, { useState, useEffect } from 'react';
-import {BrowserRouter as Router, Route, Switch} from 'react-router-dom';
+import { useState, useEffect } from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 import { getArticles, getSentiment } from '../../apiCalls';
 import { CleanedArticle } from '../../Models';
-import Form from '../Form/Form';
 import Feed from '../Feed/Feed';
-import Article from '../Article/Article';
 import Header from '../Header/Header';
-import TopicForm from '../TopicForm/TopicForm';
-import TopicFeed from '../TopicFeed/TopicFeed'
+import History from '../History/History'
 import NoMatch from '../NoMatch/NoMatch';
+import SentimentForm from '../SentimentForm/SentimentForm';
+import TopicForm from '../TopicForm/TopicForm';
 import './App.css';
-import History from '../History/History'
 
 const App = (): JSX.Element => {
   const [articles, setArticles] = useState<CleanedArticle[]>([]);
-  const [unreadArticles, setUnreadArticles] = useState<CleanedArticle[] | undefined>([])
+  const [unreadArticles, setUnreadArticles] = useState<
```

Comment:
Is this spacing because of the line length?

Reply:
Yes.